File: cpp/what-is-grit.cpp
#include <iostream>

double grit_score(double perseverance_effort, double consistency_interest) {
    return 0.60 * perseverance_effort + 0.40 * consistency_interest;
}

int main() {
    std::cout << "Synthetic grit score: "
              << grit_score(0.75, 0.50)
              << std::endl;
    return 0;
}

File: go/what-is-grit.go
package main

import "fmt"

func GritScore(perseveranceEffort float64, consistencyInterest float64) float64 {
	return 0.60*perseveranceEffort + 0.40*consistencyInterest
}

func main() {
	fmt.Printf("Synthetic grit score: %.3f\n", GritScore(0.75, 0.50))
}
